src/main/java/org/jabref/gui/ai/components/aichat/AiChatComponent.java

```diff
@@ -41,6 +45,10 @@ public class AiChatComponent extends VBox {
     private final TaskExecutor taskExecutor;
 
     private final IntegerProperty blockScroll = new SimpleIntegerProperty(0);
+    // -1 for empty, new, or non-existent messages.
+    // Is user scrolls for messages, then it will be non-negative.
+    private final IntegerProperty currentUserMessageScroll = new SimpleIntegerProperty(-1);
```

Comment:
Introduce a constant for `-1`: `NEW_EMPTY_NON_EXISTENT`.

Code should not contain magic numbers. With a reference to the constant ( `private static final`), it is possible to trace the constant.

This also gets rid of comment line 48.
line 49 can be kept